src/applications/mhv/secure-messaging/tests/e2e/secure-messaging-navigate-away-from-replying-message.cypress.spec.js

```diff
@@ -6,7 +6,7 @@ describe('verify navigate away pop-up message', () => {
   const landingPage = new PatientInboxPage();
   const site = new SecureMessagingSite();
 
-  it('navigate away by click on the inside link', () => {
+  it.skip('navigate away by click on the inside link', () => {
```

Comment:
are we able to have this resolved with test engineers prior to merging to avoid skipping?